.gitignore

```diff
@@ -8,8 +8,8 @@ umoci
 skopeo
 
 # Binaries
-agent-protocol-forwarder
-cloud-api-adaptor
+/agent-protocol-forwarder
```

Comment:
Could you revert this change as it will no longer ignore binaries in sub-folders?

Reply:
OK. I'll revert his.

Reply:
done